vars/rollingUpgradePipeline.groovy

```diff
@@ -62,7 +62,7 @@ def call(Map pipelineParams) {
                                             wrap([$class: 'BuildUser']) {
                                                 dir('scylla-cluster-tests') {
                                                     checkout scm
-                                                    try {
+                                                    catchError(stageResult: 'FAILURE') {
```

Comment:
what's the different between the try...catch   and catchError()?

If test failed, and collect log succeeded, will the test status will be treated as job status?